src/bin/pg_autoctl/primary_standby.c

```diff
@@ -327,6 +327,33 @@ postgres_add_default_settings(LocalPostgresServer *postgres)
 }
 
 
+/*
+ * primary_create_user creates a user.
+ */
+bool
+primary_create_user(LocalPostgresServer *postgres, char *userName,
+					char *password, char *hostname, char *authMethod)
```

Comment:
Is it really necessary to add an intermediate function here rather than calling `pgsql_create_user` directly. On a quick reading I don't see much logic added? Maybe it's worth it for the `pgsql_finish` call only, but we could certainly handle that at the caller level?

Reply:
Sounds reasonable.